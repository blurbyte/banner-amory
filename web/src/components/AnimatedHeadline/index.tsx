import styled from 'styled-components';

import { fadeIn } from '@styles/animations';
import BaseHeadline from '../Headline';

const AnimatedHeadline = styled(BaseHeadline)`
  animation: ${fadeIn} 400ms ease-in forwards;
`;

export default AnimatedHeadline;
