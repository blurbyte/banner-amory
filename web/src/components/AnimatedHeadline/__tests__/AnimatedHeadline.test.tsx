import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import AnimatedHeadline from '../index';

test('renders correctly', () => {
  const component = renderer.create(<AnimatedHeadline />);
  expect(component.toJSON()).toMatchSnapshot();
});
