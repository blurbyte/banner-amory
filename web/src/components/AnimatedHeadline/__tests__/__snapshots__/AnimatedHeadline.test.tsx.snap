// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c1 {
  font-size: 3rem;
  font-weight: 600;
  margin: 0;
  margin-bottom: 1.2rem;
}

.c0 {
  -webkit-animation: eMLfYp 400ms ease-in forwards;
  animation: eMLfYp 400ms ease-in forwards;
}

<h1
  className="c0 c1"
/>
`;
